smartcontactmanager/src/main/java/com/scm/smartcontactmanager/entity/contactcategoryrelation/ContactCategoryRelation.java

```diff
@@ -0,0 +1,36 @@
+package com.scm.smartcontactmanager.entity.contactcategoryrelation;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+@Entity
+@Table(name = "contact_category_relation")
+@Data
+@RequiredArgsConstructor
+@SuperBuilder
+public class ContactCategoryRelation {
+	
```

Comment:
Remove extra new lines

Reply:
The issue have been resolved. Please check it once.